src/components/MDXComponents/MDXLink.tsx

```diff
@@ -1,11 +1,14 @@
 import Link from 'next/link';
 import ExternalLink from '@/components/ExternalLink';
+import { useCurrentPlatform } from '@/utils/useCurrentPlatform';
 
 export const MDXLink = (props) => {
   const { href, children } = props;
   const isInternal = href && (href.startsWith('/') || href.startsWith('#'));
+  const currentPlatform = useCurrentPlatform();
+  const updatedHref = href.replace('%5Bplatform%5D', currentPlatform);
```

Comment:
Had to use `%5Bplatform%5D` here because every time I was debugging this step it doesn't show `[platform]` with the square braces and instead encodes them. Does anyone have thoughts on this?